skypy/galaxy/redshift.py

```diff
@@ -249,3 +250,160 @@ def herbel_pdf(redshift, alpha, a_phi, b_phi, a_m, b_m, cosmology,
     gx = np.fabs(special.gammaincc(alpha+1, x))
     pdf = dv * b_phi * np.exp(a_phi * redshift + lg) * gx
     return pdf
+
+
+def peng_redshift(alpha, a_phi, b_phi, m_star, cosmology, low=0.0, high=2.0,
+                  size=None, m_min=10**7, resolution=100):
+    r""" Redshift following the Schechter luminosity function marginalised over
+    luminosities following the Herbel et al. [1]_ model.
+
+    Parameters
+    ----------
+    alpha : float or scalar
+        The alpha parameter in the Schechter stellar mass function
+    a_phi, b_phi : float or scalar
+        Slope and intercept of the linear parametrisation of Phi_*.
+    m_star : float or int
+        Characteristic stellar mass M_*.
+    cosmology : instance
+        Instance of an Astropy Cosmology class.
+    low : float or array_like of floats, optional
```

Comment:
Can you specify that this if the low boundary for redshift? Also, I do not see why we want a default value here